neuralprophet/wrapper.py

```diff
@@ -1,8 +1,73 @@
 from neuralprophet.forecaster import NeuralProphet
 import pandas as pd
+import logging
+
+log = logging.getLogger("NP.forecaster")
 
 
 class Prophet(NeuralProphet):
+    def __init__(
+        self,
+        growth="linear",
+        changepoints=None,
+        n_changepoints=25,
+        changepoint_range=0.8,
+        yearly_seasonality="auto",
+        weekly_seasonality="auto",
+        daily_seasonality="auto",
+        holidays=None,
+        seasonality_mode="additive",
+        seasonality_prior_scale=10.0,
+        holidays_prior_scale=10.0,
+        changepoint_prior_scale=0.05,
+        mcmc_samples=0,
+        interval_width=0.80,
+        uncertainty_samples=1000,
+        stan_backend=None,
+    ):
+        # holidays, seasonality_prior_scale, holidays_prior_scale,
+        # changepoint_prior_scale, mcmc_samples, interval_width, uncertainty_samples, stan_backend
+        # Set Prophet-like default args
+        kwargs["quantiles"] = [0.9, 0.1]
```

Comment:
you can map `interval_width` to `prediction_interval`, see https://github.com/ourownstory/neural_prophet/pull/715/